ScreamingUdder/mantid
# 37 off instrument geometry

Have created an algorithm (LoadNexusGeometry) which creates an empty instrument and adds it to the instrument data service.

There are also member functions for adding a source, sample, detector and generic component to the instrument.

**To test:**

Build target DataHandlingTest.
DataHandlingTest_LoadNexusGeometry should pass without failure.

There are three tests, which check initialization, execution, and the operation of the member functions. 

**Release Notes** 

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
